CHANGES

```diff
@@ -33,6 +33,20 @@
 # New collective algorithms. All collective algorithms are listed in
   src/mpi/coll/coll_algorithms.txt
 
+# Removed hydra2.
```

Comment:
We should add something here about Hydra2 features being ported to Hydra.

Reply:
Good suggestion! done.